src/components/Trailers/api/services/getTrailersData.ts

```diff
@@ -0,0 +1,5 @@
+import { cardData } from "../../../../mocks/cardData";
+
+export const getTrailersData = () => (
+    cardData.slice(0,4)
+);
```

Comment:
your function should be independent. So, an array should be passed as an argument

Reply:
dona